### PR TITLE
chore: Replace token names with their addresses

### PR DESCRIPTION
**Pull request by Chef-Chungus**

### CODE REVIEW

File: src/config/constants/tokens.ts [hunk 1]
```diff
@@ -780,25 +780,25 @@ const tokens = {
   yfii: {
     symbol: 'YFII',
     address: {
+      56: '0x7F70642d88cf1C4a3a7abb072B53B929b653edA5',
       97: '0xE02dF9e3e622DeBdD69fb838bB799E3F168902c5',
-      56: '0x7f70642d88cf1c4a3a7abb072b53b929b653eda5',
```
Comment (Chef-Chungus) [inline src/config/constants/tokens.ts:784]: The ordering of the testnet and mainnet addresses was wrong.

File: src/config/constants/tokens.ts [hunk 2]
```diff
@@ -780,25 +780,25 @@ const tokens = {
...
     },
     decimals: 18,
     projectLink: 'https://dfi.money/#/',
   },
   xtz: {
     symbol: 'XTZ',
     address: {
+      56: '0x16939ef78684453bfDFb47825F8a5F714f12623a',
       97: '0xE02dF9e3e622DeBdD69fb838bB799E3F168902c5',
-      56: '0x16939ef78684453bfdfb47825f8a5f714f12623a',
```
Comment (Chef-Chungus) [inline src/config/constants/tokens.ts:793]: The ordering of the testnet and mainnet addresses was wrong.

File: src/config/constants/tokens.ts [hunk 3]
```diff
@@ -744,16 +744,16 @@ const tokens = {
   usdt: {
     symbol: 'USDT',
     address: {
+      56: '0x55d398326f99059fF775485246999027B3197955',
       97: '0xE02dF9e3e622DeBdD69fb838bB799E3F168902c5',
-      56: '0x55d398326f99059ff775485246999027b3197955',
```
Comment (Chef-Chungus) [inline src/config/constants/tokens.ts:748]: The ordering of the testnet and mainnet addresses was wrong.

File: src/config/constants/tokens.ts [hunk 4]
```diff
@@ -835,15 +835,15 @@ const tokens = {
     symbol: 'BAKE',
     address: {
       56: '0xE02dF9e3e622DeBdD69fb838bB799E3F168902c5',
-      97: '0xE02dF9e3e622DeBdD69fb838bB799E3F168902c5',
```
Comment (Chef-Chungus) [inline src/config/constants/tokens.ts:838]: The testnet address was incorrect

**Review by hachiojidev — APPROVED** [review]

**Review by aldoromo88 — APPROVED** [review]

**Pull request merged**